fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/JobsServletSpec.groovy

```diff
@@ -155,6 +159,36 @@ class JobsServletSpec extends Specification {
         GroovyTestUtils.compareJson(result, expectedResponse, JsonSortStrategy.SORT_BOTH)
     }
 
+    def "jobs endpoint returns filtered results when a valid filter is present"() {
+        setup:
+        String expectedResponse = """{"jobs":[
+                                        {
+                                            "dateCreated":"2016-01-01",
+                                            "jobTicket":"ticket1",
+                                            "query":"https://localhost:9998/v1/data/QUERY",
+                                            "results":"http://localhost:9998/jobs/ticket1/results",
+                                            "self":"http://localhost:9998/jobs/ticket1",
+                                            "status":"success",
+                                            "syncResults":"http://localhost:9998/jobs/ticket1/results?asyncAfter=never",
+                                            "userId": "momo"
+                                        }
+                                  ]}"""
+
+        when: "We send a request to the /jobs endpoint with a valid filters parameter"
+        String result = makeRequest("/jobs", [filters : ["userId-eq[momo]"]])
+
+        then: "We only get the job payload that satisfies the filter"
+        GroovyTestUtils.compareJson(result, expectedResponse, JsonSortStrategy.SORT_BOTH)
+    }
+
+    def "jobs endpoint returns an empty list when none of the JobRows satisfy the filter"() {
+        when: "We send a request to the /jobs endpoint with a valid filters parameter"
+        String result = makeRequest("/jobs", [filters : ["userId-eq[pikachu]"]])
+
+        then: "We only get the job payload that satisfies the filter"
+        GroovyTestUtils.compareJson(result, "{\"jobs\":[]}", JsonSortStrategy.SORT_BOTH)
```

Comment:
If you use single quotes to delimitate the string, you don't have to escape the double quotes around `jobs`.